### DISCUSSION TITLE
Any plan for testing (unit/E2E test)?

### DISCUSSION THREAD

**Discussion by berviantoleo** [dkhd/winterly-tab#26] [Ideas]
Hi, do you have any plans for E2E test or unit test? I think it will be helpful for the development. Any idea for this? Like what we need to use.

**Comment by dkhd** [comment]
Hi @berviantoleo 👋

Sorry for quite late response, occupied by IRL actitivies 😄

Currently, we still have to unit testing, but it would be great if someone could help. If you want to lend a hand for testing, we are more than welcome! 😊

Salam..

**Reply by berviantoleo** [reply]
Yes sure.

**Reply by dkhd** [reply]
Thank you 🔥🔥🔥
